src/openrct2/ride/Vehicle.h

```diff
@@ -181,7 +181,7 @@ struct Vehicle : SpriteBase
     };
     uint8_t animation_frame;
     uint8_t pad_C6[0x2];
-    uint16_t var_C8;
+    uint16_t animation_state;
```

Comment:
We are in the process of renaming everything to `camelCase`. As such, the appropriate name would be `animationState`.